Add OpenTicket backup instructions to update procedure (#3838)
centreon/centreon-documentation: versioned_docs/version-23.04/alerts-notifications/ticketing-install.md

Your Centreon Open Tickets Module is now installed.

![image](../assets/alerts/open_tickets_install_02.png)

## Creating a backup

To create a backup of your Open Ticket provider configuration, follow these steps:

* Make a backup of these folders: **/usr/share/centreon/www/modules/centreon-open-tickets** and **/usr/share/centreon/www/widgets/open-tickets**
* After updating, copy paste the backup **register.php** file(s) to **/usr/share/centreon/www/modules/centreon-open-tickets/providers/**

> It is particularly important to create a backup of your configuration before [updating Centreon](../update/update-centreon-platform.md) to avoid them being erased during the update process.

Author: ```suggestion
It is particularly important to create a backup of your configuration before [updating Centreon](../update/update-centreon-platform.md) to avoid them being erased during the update process.

```

Your Centreon Open Tickets Module is now installed.

![image](../assets/alerts/open_tickets_install_02.png)

> ## Creating a backup

Reviewer: ```suggestion
## Creating a backup of your custom Open Ticket provider configurations
```

> To create a backup of your Open Ticket provider configuration, follow these steps:
> 
> * Make a backup of these folders: **/usr/share/centreon/www/modules/centreon-open-tickets** and **/usr/share/centreon/www/widgets/open-tickets**
> * After updating, copy paste the backup **register.php** file(s) to **/usr/share/centreon/www/modules/centreon-open-tickets/providers/**
> 
> It is particularly important to create a backup of your configuration before [updating Centreon](../update/update-centreon-platform.md) to avoid them being erased during the update process.

Reviewer: ```suggestion
If you need to [update Centreon](../update/update-centreon-platform.md), first backup your custom Open Ticket provider configurations so as not to lose them.

* Make a backup of these folders: **/usr/share/centreon/www/modules/centreon-open-tickets** and **/usr/share/centreon/www/widgets/open-tickets**.
* After updating, copy and paste the backup **register.php** file(s) to **/usr/share/centreon/www/modules/centreon-open-tickets/providers/**.

```
